crates/interpreter/src/valid.rs

```diff
@@ -558,18 +560,22 @@ impl<'a, 'm> Expr<'a, 'm> {
             Nop => (),
             Block(b) => self.push_label(self.blocktype(&b)?, LabelKind::Block)?,
             Loop(b) => {
-                self.push_label(self.blocktype(&b)?, LabelKind::Loop(self.branch_target()))?
+                let branch = self.branch_target();
+                self.push_label(self.blocktype(&b)?, LabelKind::Loop(branch))?
```

Comment:
What is the rust version used by the CI? I use `rustc 1.82.0-nightly (28a58f2fa 2024-07-31)`, and there is a borrow checker error for me.